runtimes/runtimes/util/loggingUtil.ts

```diff
@@ -0,0 +1,68 @@
+import { Logging } from '../../server-interface'
+import { Connection } from 'vscode-languageserver/node'
+
+export type LogLevel = 'error' | 'warn' | 'info' | 'log' | 'debug'
+
+const logLevelMap: Record<LogLevel, number> = {
+    error: 0,
+    warn: 1,
+    info: 2,
+    log: 3,
+    debug: 4,
+}
+
+export const isLogLevelEnabled = (l1: LogLevel, l2: LogLevel): boolean => {
+    return logLevelMap[l1] >= logLevelMap[l2]
+}
+
+export const isValidLogLevel = (level: LogLevel): boolean => {
+    return level && typeof level === 'string' && ['error', 'warn', 'info', 'log', 'debug'].includes(level)
+}
+
+const DEFAULT_LOG_LEVEL: LogLevel = 'log'
+
+export const getLogLevel = async (lspConnection: Connection): Promise<LogLevel> => {
+    const logLevelConfig = await lspConnection.workspace.getConfiguration({
+        section: 'logLevel',
```

Comment:
Let prepend section with our own prefix to avoid accidental collision with other configs. We can use `aws.logLevel` here to scope this down.